initializr-service/src/main/resources/application.yml

```diff
@@ -1256,6 +1263,17 @@ initializr:
           links:
            - rel: reference
              href: https://docs.spring.io/spring-shell/docs/2.0.0.M2/reference/htmlsingle/
+        - name: Spring Statemachine
+          id: spring-statemachine
+          groupId: org.springframework.statemachine
+          artifactId: spring-statemachine-starter
+          description: Statemachine integration
+          version: 2.0.0.M3
+          versionRange: 2.0.0.RC1
+          bom: spring-statemachine
+          links:
+            - rel: reference
+              href: http://docs.spring.io/spring-statemachine/docs/2.0.0.M3/reference/htmlsingle/
```

Comment:
Can we have a link without the version please (something like `current`).

Reply:
Can we also have a reference to a sample that represents a simple project as created from start.spring.io with a typical usage of the project?

Reply:
I can probably put these into docs for next milestone/rc. I had a plan to create a new simple bare sample.